DataFormats/CaloTowers/src/classes_def.xml

```diff
@@ -3,7 +3,9 @@
    <version ClassVersion="11" checksum="4063982642"/>
    <version ClassVersion="10" checksum="2444816232"/>
   </class>
-  <class name="CaloTower" ClassVersion="12">
+  <class name="CaloTower" ClassVersion="14">
+   <version ClassVersion="14" checksum="2436724497"/>
+   <version ClassVersion="13" checksum="3299778598"/>
```

Comment:
we should not keep transient version numbers.
Please remove the current v13 (checksum="3299778598"); and change the current v14 to 13 (no skip in version numbers)